nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -371,7 +371,8 @@ in {
       "autofs"
       # systemd-cryptenroll
     ] ++ lib.optional cfg.enableTpm2 "tpm-tis"
-    ++ lib.optional (cfg.enableTpm2 && !(pkgs.stdenv.hostPlatform.isRiscV64 || pkgs.stdenv.hostPlatform.isArmv7)) "tpm-crb";
+    ++ lib.optional (cfg.enableTpm2 && !(pkgs.stdenv.hostPlatform.isRiscV64 || pkgs.stdenv.hostPlatform.isArmv7)) "tpm-crb"
+    ++ lib.optional cfg.package.withEfi "efivarfs";
```

Comment:
I use [`fs-efivarfs`](https://github.com/philiptaron/nix-config/blob/lex/boot.nix#L44) so I don't know if this actually worked.

Reply:
`fs-efivarfs` is an alias of `efivarfs`.